Cartfile

```diff
@@ -1 +1 @@
-github "ReactiveCocoa/ReactiveCocoa" "v3.0-beta.1"
+github "ReactiveCocoa/ReactiveCocoa" ~> 3.0
```

Comment:
If this works for betas, it's purely by happenstance. /cc https://github.com/Carthage/Carthage/issues/469#issuecomment-111931972

I think we should pin explicitly to the latest beta instead.